Stop notification_helper and snoretoast from getting shimmed

Starz0r opened the pull request:
You have some unnecessary files getting shimmed on install, this PR is here to stop that from happening.

Starz0r committed:
Stop notification_helper and snoretoast from getting shimmed

Starz0r committed:
Create ignore files before install or they won't actually get ignored

scowalt commented:
Thanks for catching this and for the contribution! Can you update the package version to `1.11.0.20201229` with this change? Then I can publish directly from this PR once it's merged in

Starz0r committed:
Create folders leading to snoretoast and silence New-Item output

Starz0r committed:
snoretoast exists in a different directory on 32-bit platforms

Starz0r committed:
v1.11.0.20201229 fix

scowalt commented on Streamlink Twitch GUI/tools/chocolateyinstall.ps1:
```diff
@@ -4,6 +4,13 @@ $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 $installDir = Split-Path -parent $toolsDir
 $downloadPath = "https://github.com/streamlink/streamlink-twitch-gui/releases/download/$packageVersion/"
 
+New-Item -ItemType file $installDir\$packageName\notification_helper.exe.ignore -EA 0 | Out-Null
```
I've tried this out locally and I'm still seeing `notification_helper.exe` get shim'd into `C:\ProgramData\chocolatey\bin`. I assume this should be fixed as a part of this PR? (Or maybe there's something I'm missing)

Starz0r replied:
What the hell, I literally included it. Looks like [`3e9885d` (#41)](https://github.com/streamlink/streamlink-twitch-gui-chocolatey/pull/41/commits/3e9885d8859b7dd5c8d412003e1eb9e1a5c958b6) pummeled it somehow.

Starz0r replied:
Deleted my last comment, turns out that line does exist locally, going to investigate more.

Starz0r replied:
Resolved by [`318cded` (#41)](https://github.com/streamlink/streamlink-twitch-gui-chocolatey/pull/41/commits/318cdeddaf0cb301013d4f247df8b20956dc30de)

Starz0r committed:
Install directory needs to exist beforehand

Looks like New-Item is also recursive with directories, so this probably can be simplified.

Starz0r reviewed (commented):
Pretty sure this solves it. Looks like if `-ItemType` is `file` then no directories are created, but an `-ItemType` of `directory` recursively creates directories, which is why that is the only line that fails.

scowalt commented:
Thanks again for the contribution. This change is in chocolatey's moderation queue. I'm not sure how this will impact folks who are upgrading, but everyone on a fresh install will get this fix.
